gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -288,6 +331,295 @@ gst_tensor_src_iio_init (GstTensorSrcIIO * self)
   return;
 }
 
+/**
+ * @brief check if device/trigger with the given name exists
+ * @return >=0 if OK, represents device/trigger number
+ *         -1  if returned with error
+ */
+static guint
+gst_tensor_src_iio_get_id_by_name (const gchar * dir_name, const gchar * name,
+    const gchar * prefix)
+{
+  DIR *dptr = NULL;
+  GError *error = NULL;
+  struct dirent *dir_entry;
+  gchar *filename = NULL;
+  guint id = -1;
+  gchar *file_contents = NULL;
+  guint ret = -1;
```

Comment:
-1 for unsigned int?

Reply:
Corrected here as well as at other places.